diag_manager/diag_manager.F90

```diff
@@ -1473,6 +1473,7 @@ LOGICAL FUNCTION send_data_3d(diag_field_id, field, time, is_in, js_in, ks_in, &
                                ie_in=ie_in, je_in=je_in, ke_in=ke_in, weight=weight, err_msg=err_msg)
     endif
   END FUNCTION send_data_3d
+
```

Comment:
Is it ok if we get rid of this extra line?

Reply:
yeah must've snuck in when i fixed the conflict, should be fixed now